site/content/docs/0.0/layout/gutters.md

```diff
@@ -10,41 +10,18 @@ toc: true
 
 ## How they work
 
-- **Gutters are the gaps between column content, created by horizontal `padding`.** We set `padding-right` and `padding-left` on each column, and use negative `margin` to offset that at the start and end of each row to align content.
+- **Gutters are the gaps between column content, created by horizontal `padding`.** We set `padding-right` and `padding-left` on each column, and use negative `margin` to offset that at the start and end of each row to align content. The default gutter width is described in [our grid options]({{< docsref "/layout/grid#grid-options" >}}).
 
-- **Gutters start at `2px` and go up to `64px` wide.** This allows us to match our grid to the [padding and margin spacers]({{< docsref "/utilities/spacing" >}}) scale.
+- **Gutter utilities start at `2px` and go up to `64px` wide.** This allows us to match our grid to the [padding and margin spacers]({{< docsref "/utilities/spacing" >}}) scale.
```

Comment:
Gutters was fine, wasn't it?

```suggestion
- **Gutters start at `2px` and go up to `64px` wide.** This allows us to match our grid to the [padding and margin spacers]({{< docsref "/utilities/spacing" >}}) scale.
```

Reply:
Quite not sure about this change, it looks fine to me right now.

Reply:
It's just that Bootstrap uses "Gutters" in their wording. As you want. I'll let you decide.